src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -121,6 +121,7 @@
       <_Dependency Remove="Microsoft.Win32.Registry"/>
       <_Dependency Remove="System.Security.AccessControl"/>
       <_Dependency Remove="System.Security.Principal.Windows"/>
+      <_Dependency Remove="Microsoft.Bcl.AsyncInterfaces"/>
```

Comment:
@tmat can you sign off on this bit?  @jasonmalinowski thinks this is the way to go, but isn't sure